Sources/TensorFlow/Layers/Upsampling.swift

```diff
@@ -108,9 +108,6 @@ public struct UpSampling3D<Scalar: TensorFlowFloatingPoint>: Layer {
     /// - Returns: The output.
     @differentiable
     public func call(_ input: Tensor<Scalar>) -> Tensor<Scalar> {
-        let shape = input.shape
```

Comment:
This seems to have been already deleted @eaplatanios

Reply:
Oh this pops up in all the PRs, probably because of the matmul enhancement commit.

Reply:
Yes, that's right. We're going through the PRs in order and will merge with master before checking them in. :)